### PR TITLE
367: validation for Send To message functionality

### PR DESCRIPTION
**Pull request by rachaelparris**

### CODE REVIEW

File: web-client/src/views/StartCase/Contacts.jsx [hunk 1]
```diff
@@ -7,12 +7,12 @@ import { ContactSecondary } from './ContactSecondary';
 
 export const Contacts = connect(
   {
```
Comment (kkoskelin) [inline web-client/src/views/StartCase/Contacts.jsx:9]: @ericsorenson any thought on whether it'd be okay to simply `{...props}` ?

**Review by kkoskelin — APPROVED** [review]

File: web-client/src/views/StartCase/Contacts.jsx (hunk 1 above)
Comment (ericsorenson) [reply web-client/src/views/StartCase/Contacts.jsx:9]: @kkoskelin that should work...

Comment (rachaelparris) [reply web-client/src/views/StartCase/Contacts.jsx:9]: It seems like we're doing this differently in different places. CreateMessageModalDialog.jsx is an example. Maybe we need to standardize it?

**Pull request merged**